numpy/typing/tests/data/reveal/ufuncs.pyi

```diff
@@ -74,3 +75,14 @@ assert_type(np.matmul.signature, Literal["(n?,k),(k,m?)->(n?,m?)"])
 assert_type(np.matmul.identity, None)
 assert_type(np.matmul(AR_f8, AR_f8), Any)
 assert_type(np.matmul(AR_f8, AR_f8, axes=[(0, 1), (0, 1), (0, 1)]), Any)
+
+assert_type(np.bitwise_count.__name__, Literal['bitwise_count'])
+assert_type(np.bitwise_count.ntypes, Literal[11])
+assert_type(np.bitwise_count.identity, None)
+assert_type(np.bitwise_count.nin, Literal[1])
+assert_type(np.bitwise_count.nout, Literal[1])
+assert_type(np.bitwise_count.nargs, Literal[2])
+assert_type(np.bitwise_count.signature, None)
+assert_type(np.bitwise_count.identity, None)
+assert_type(np.bitwise_count(i8), Any)
+assert_type(np.bitwise_count(AR_i8), Any)
```

Comment:
This is failing now. I'll take a look tomorrow.

Reply:
I think `npt.NDArray[Any]` is correct?

Reply:
Correct, I'm somewhat surprised that it even produced an `Any` here at one point. Though it could have been accepted due to the somewhat broad substring matching used previously.

Reply:
Ah I see. That makes sense why it was passing before.